src/objective-c/GRPCClient/private/NSDictionary+GRPC.h

```diff
@@ -32,9 +32,9 @@
  */
 
 #import <Foundation/Foundation.h>
-
-struct grpc_metadata;
+#include <grpc/grpc.h>
 
 @interface NSDictionary (GRPC)
 + (instancetype)grpc_dictionaryFromMetadata:(struct grpc_metadata *)entries count:(size_t)count;
+- (size_t)grpc_toMetadataArray:(grpc_metadata **)metadata;
```

Comment:
Methods that return data indirectly via pointers provided by the caller are named getX:y:z: (check https://developer.apple.com/library/mac/documentation/Cocoa/Conceptual/CodingGuidelines/Articles/NamingMethods.html )
So this one should be:
- grpc_getMetadataArray:(grpc_metadata **)metadata count:(size_t *)count;

Although NSDictionary already has a count property, so returning it here seems redundant.